x-pack/legacy/plugins/maps/public/layers/styles/vector_style_defaults.js

```diff
@@ -11,6 +11,7 @@ import { COLOR_GRADIENTS } from './color_utils';
 const DEFAULT_COLORS = ['#e6194b', '#3cb44b', '#ffe119', '#f58231', '#911eb4'];
 const DEFAULT_ICON = 'airfield';
 
+export const ABSOLUTE_MIN_SIZE = 0;
 export const DEFAULT_MIN_SIZE = 1;
```

Comment:
Maybe we should consider changing these names. They're both defaults, just different ones. Maybe something like `DEFAULT_TO_NOTHING` and `DEFAULT_TO_MINIMAL`